app/views/payment/v2/StudentLicensesComponent.vue

```diff
@@ -73,7 +73,7 @@ export default {
     await this.fetchTeacherPrepaids({ teacherId: me.get('_id') })
     const prepaids = this.teacherPrepaids(me.get('_id'))
     // not including expired license in count since we don't show them in UI so it will be confusing
-    if ((prepaids && ((prepaids.pending.length + prepaids.empty.length + prepaids.available.length) > 0)) || ['cambodia', 'viet-nam'].includes(me.get('country'))) {
+    if ((prepaids && ((prepaids.pending.length + prepaids.empty.length + prepaids.available.length) > 0)) || ['cambodia', 'viet-nam', 'japan'].includes(me.get('country'))) {
```

Comment:
Does this work for Vietnam? I would think it would be `'vietnam'` instead of `'viet-nam'`.

Actually, not sure why we have both in database, but there are 23x as many users from `'vietnam'` as there are from `'viet-nam'`, and I think `'viet-nam'` is a bug that is getting in there somehow (shouldn't be in our countries list).
```suggestion
    if ((prepaids && ((prepaids.pending.length + prepaids.empty.length + prepaids.available.length) > 0)) || ['cambodia', 'viet-nam', 'vietnam', 'japan'].includes(me.get('country'))) {
```